### ISSUE TITLE
New logo/icon proposal

### ISSUE THREAD

**Issue by desuto** [DarkionAvey/AchievementUnlocked-library#6]
Hi, I am a graphic designer, I want to help others in graphic design.

After I reviewed your project, your logo on this project not interesting. Therefore I want to contribute to this project by creating a new logo / icon. what do you think?

**Comment by DarkionAvey** [comment]
Sure, I'd love to see it. 
While you're at it, please make the icon compatible with adaptive icons. 
An adaptive icon is basically an icon that has foreground layer (for example: the logo text) and background layer (solid color, gradient, image, etc)

**Comment by desuto** [comment]
yes. I will design it. You can see it in a moment

**Comment by desuto** [comment]
I have design logo for this project. what do you think ? do you like it ? wich one do you lilke it ?

![achievement unlocked library](https://user-images.githubusercontent.com/39637790/43594296-1913d8f8-96a4-11e8-8fe8-0c38c379bfc7.png)

**Comment by DarkionAvey** [comment]
Having a lock symbol is pretty cool. You could remove the AU text, it's not necessary. Maybe merge the lock with a horizontal popup, since the app is basically horizontal popups? We can pick a suitable color later

**Comment by desuto** [comment]
Hi sir, did you mean like this ?

![achievement unlocked library](https://user-images.githubusercontent.com/39637790/43671477-10dc0116-97c5-11e8-90b0-93dc1a1a6c27.png)

**Comment by DarkionAvey** [comment]
Yes, that's cool. When you feel like it, please post the PSD file so I can edit it to match Android's standards
